### PR TITLE
Jr/initial tpm2 util

### LINKED ISSUE #7: TPM2 PCR value on client side
vytáhnout PCR values z modulu a nějak je umět použít do naší appky

### PR DESCRIPTION
**Pull request by xrosch**
Initial functionality for TPM2. Includes functions for communicating (e.g., get random bytes, get PCR values). Also features a demo to quickly test TPM2 functionality.

### CODE REVIEW

File: tpm2_util.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+from TPM2.Tpm import *
+
+
+def get_random_bytes(tpm: Tpm, n: int) -> list[int]:
```
Comment (T0mmas) [inline tpm2_util.py]: I've got an error here `TypeError: 'type' object is not subscriptable`.
Maybe it is needed to use `List` from `typing`

File: tpm2_util.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+def get_property_vendor_string(tpm: Tpm) -> str:
+    """Get vendor string 1 from the TPM."""
+    cap_res = tpm.GetCapability(TPM_CAP.TPM_PROPERTIES, TPM_PT.VENDOR_STRING_1, 1)
+    string = cap_res.capabilityData.tpmProperty[0].value
+
+    res_bytes = string.to_bytes(int(string.bit_length() / 8 + 0.5), "big")
+    return "".join([chr(i) for i in res_bytes])
+
+
+def get_pcr_values(tpm: Tpm, alg: TPM_ALG_ID = TPM_ALG_ID.SHA1) -> list[bytearray]:
```
Comment (T0mmas) [inline tpm2_util.py]: Same here for me, error `TypeError: 'type' object is not subscriptable`

**Review by T0mmas — DISMISSED** [review]

File: tpm2_util.py (hunk 1 above)
Comment (Matej4545) [reply tpm2_util.py]: I can run it without errors. Linux specific issue maybe?

Comment (T0mmas) [reply tpm2_util.py]: Or python version maybe 😄

**Review by Matej4545 — APPROVED** [review]
On Windows everything works for me.

**Review by T0mmas — APPROVED** [review]

**Pull request merged**